packages/dropdownMenu/components/DropdownMenuItem.tsx

```diff
@@ -9,12 +10,49 @@ export interface DropdownMenuItemProps {
    * The value that the item represents
    */
   value: string;
+  /**
+   * Whether to allow other actions on the item
+   */
   disabled?: boolean;
+  /**
+   * The length of all DropdownMenuItems in its parent DropdownMenuSection
+   */
+  listLength?: number;
+  /**
+   * The index within the DropdownMenuSection
+   */
+  index?: number;
+  /**
+   * If the current item is in use
+   */
+  isActive?: boolean;
+  /**
+   * Children for PopoverListItem
+   */
   children: React.ReactNode;
```

Comment:
❓  Do you think it would be helpful to add a configurable `data-cy` here? It looks like there is only one set for the Popover items.